dao-services/src/main/resources/db/changelog/wiki.db.changelog-1.0.0.xml

```diff
@@ -98,6 +98,14 @@
     </modifySql>
   </changeSet>
 
+  <changeSet author="wiki" id="1.0.0-3.1">
```

Comment:
Why adding this changeset and the changeset 1.0.0-5.1 in the middle of previous changesets and not just before the migration tag ?